Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -364,10 +364,13 @@ class StringSetting: Setting, UITextFieldDelegate {
         cell.accessibilityTraits = UIAccessibilityTraits.none
         cell.contentView.addSubview(textField)
 
+        textField.font = DynamicFontHelper.defaultHelper.DefaultStandardFont
```

Comment:
This is the only change that is needed. Changing the constraints has caused the height of the row to be incorrect.

Reply:
Okay, the changes to the constraints have been reverted!